core/search/levenshtein_filter.hpp

```diff
@@ -61,16 +61,21 @@ struct IRESEARCH_API by_edit_distance_filter_options {
   //////////////////////////////////////////////////////////////////////////////
   bool with_transpositions{false};
 
+  //////////////////////////////////////////////////////////////////////////////
+  /// @brief compute levenshtein without prefix
```

Comment:
match this number of characters from the beginning of the target regardless of edit distance